### PR TITLE
Allow PerfTests to control what ITraceManager is used

### PR DESCRIPTION
**Pull request by rchande**
The C# typing test (closed) doesn't use CPC, so it needs to be able to provide a NoOpTraceManager.

### CODE REVIEW

**Comment by rchande** [comment]
Please review @TyOverby @basoundr @KevinH-MS

**Comment by TyOverby** [comment]
LGTM

File: src/Test/Perf/Runner/Perf.Runner.csproj [hunk 1]
```diff
@@ -83,6 +81,9 @@
   <ItemGroup>
     <WCFMetadata Include="Service References\" />
   </ItemGroup>
+  <ItemGroup>
```
Comment (basoundr) [inline src/Test/Perf/Runner/Perf.Runner.csproj]: remove the empty folder?

File: src/Test/Perf/Runner/Program.cs [hunk 2]
```diff
@@ -81,11 +81,11 @@ public static void Main(string[] args)
                 testInstances.AddRange(tests);
             }
 
-            var traceManager = TraceManagerFactory.GetTraceManager();
 
-            traceManager.Initialize();
```
Comment (basoundr) [inline src/Test/Perf/Runner/Program.cs]: I am not sure what the implications of moving the `Initialize` from outside the loop to inside the loop

**Pull request merged**